python/ml4ir/applications/ranking/tests/test_metrics.py

```diff
@@ -94,3 +97,15 @@ def test_model_training(self):
         for gold_metric_name, gold_metric_val in GOLD_METRICS.items():
             assert gold_metric_name in metrics
             assert np.isclose(metrics[gold_metric_name], gold_metric_val, rtol=0.01)
+
+
+def test_top_5_categorical_accuracy():
```

Comment:
@balikasg Can probably put this in the class itself, no? I understand why you didn't. We should restructure these tests out of ranking and into the base package at some point. I think I have an issue filed for it.